endpoints.json

```diff
@@ -25,5 +25,20 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves an article with the matching input id",
+    "queries": [],
+    "exampleResponse": [
+      {
+        "title": "Living in the shadow of a great man",
+        "topic": "mitch",
+        "author": "butter_bridge",
+        "body": "I find this existence challenging",
+        "created_at": "1594329060000",
+        "votes": "100",
+        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+      }
+    ]
```

Comment:
This is correct for this endpoint! No queries for this one, eventually your /api/articles endpoint will take queries which is when you could update that one 👍